Add missing stub function to prevent underlinking error.
Patch provided by Pat Erley
See also: https://bugs.gentoo.org/show_bug.cgi?id=585994

--- a/src/dummy-session-info.c
+++ b/src/dummy-session-info.c
@@ -50,6 +50,11 @@
     return FALSE;
 }
 
+gboolean session_info_session_is_locked(struct session_info *ck)
+{
+    return FALSE;
+}
+
 gboolean session_info_is_user(struct session_info *si)
 {
     return TRUE;
